packages/bot-skeleton/src/services/api/__tests__/ticks-service.js

```diff
@@ -58,25 +58,3 @@ describe('Ticks Service', () => {
         });
     });
 });
-
-describe('Account', () => {
-    describe('Login', () => {
-        it.skip('Login should be successful', () => {
-            expect(addTokenIfValid('Xkq6oGFEHh6hJH8')).resolves;
-        });
-    });
-    describe('Login on invalid token', () => {
-        it.skip('Login should be unsuccessful', async () => {
-            try {
-                await addTokenIfValid('someinvalidtoken123xyz');
-            } catch (e) {
-                expect(e).toBeTruthy();
-            }
-        });
-    });
-    describe('logout', () => {
-        it.skip('Logout should be successful', () => {
-            expect(logoutAllTokens).not.toThrow();
-        });
-    });
-});
```

Comment:
Should we delete this?

Reply:
Since the methods, `addTokenIfValid()` and `logoutAllTokens()` are not used elsewhere, I thought its same to remove them.